libraw/src/rawimage.rs

```diff
@@ -0,0 +1,37 @@
+use crate::{Processor, Sizes};
+use std::ops::Deref;
+use std::slice;
+
+use libraw_sys as sys;
+
+pub struct RawImage {
+    _processor: Processor,
+    rawdata: *const sys::libraw_rawdata_t,
```

Comment:
Instead of storing `rawdata` we could read it from `processor.inner.rawdata` when needed. It would also make it clearer that we depend on `Processor` being there to safely access the rawdata.

Reply:
Oh, right, good idea! I got a little locked into the "RawImage has to have these members" mindset.